climada/hazard/base.py

```diff
@@ -1345,8 +1344,7 @@ def _read_att_mat(self, data, file_name, var_names):
         except ValueError as err:
             raise ValueError('Size missmatch in fraction matrix.') from err
         except KeyError:
-            self.fraction = sparse.csr_matrix(np.ones(self.intensity.shape,
-                                                      dtype=np.float))
+            self.fraction = sparse.csr_matrix(np.ones(self.intensity.shape, dtype=float))
```

Comment:
See https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated